src/main/java/seedu/allonus/expense/ExpenseTracker.java

```diff
@@ -328,6 +361,8 @@ public static void expenseRunner(TextUi ui) {
             case ("find"):
                 executeFind(rawInput);
                 break;
+            case (""):
+                break;
             default:
                 logger.log(Level.WARNING, LOG_INVALID_COMMANDS);
                 System.out.println(MSG_INVALID_COMMANDS);
```

Comment:
Is there a reason why you chose to have the empty command as a separate case, instead of as an invalid command?